### PR TITLE
Add many i18n locale translated by gpt-4o, cc #77

### PR DESCRIPTION
**Pull request by PeterDaveHello**
All missing locales missing in #77 should be included now :tada: 

## Summary by CodeRabbit

- **New Features**
  - Introduced localization support for multiple languages, enhancing the accessibility of the OpenAI API extension for users in Arabic, Bengali, Hindi, Indonesian, Malay, Russian, Thai, Tagalog, Urdu, and Vietnamese.

- **User Experience Enhancements**
  - Added comprehensive guidance for setting up the OpenAI API key, including instructions, error messages, and cost notifications in localized languages.
  - Localized prompts for functionalities like sending messages, generating code, and managing sessions, improving usability for non-English speakers.

### CODE REVIEW

**Comment by Christopher-Hayes** [comment]
Thanks Peter, I tried to do some testing.

For most of these languages, I'm not sure how people use them in VS Code. VS Code officially [only supports 15 languages](https://code.visualstudio.com/docs/getstarted/locales#_available-locales). I looked for extensions, but the store [only shows official Microsoft translations](https://marketplace.visualstudio.com/search?target=VSCode&category=Language%20Packs&sortBy=Installs), I don't see any other extension doing translations.

Regardless, I'd still like to merge these translations, even if I can't test all of them.

Based on user feedback, I think we should give users a choice in the UI about whether they want their native language or just want English. When we add that, we could also just add a UI option to turn on our custom translation for languages that VS Code does not officially support.

## Tests

Here's my testing for the languages that do show up in "Change display language":

### Russian

Russian doesn't seem to be working for me. The app still shows English.

image

Russian was the only language in this PR that is officially supported by VS Code.

**Comment by Christopher-Hayes** [comment]
Okay, the issue seems to be that VS Code uses `ru` as the Russian locale ID, but the file had `ru-ru`. When I changed it, the translation started working with VS Code. I'll push that change and it should be good to merge.

image

**Comment by Christopher-Hayes** [comment]
Actually, GitHub doesn't allow me to push to your branch.

If you could push a commit to change the Russian translation file to be `package.nls.ru.json`.

Basically what I did here: https://github.com/vscode-reborn-ai/vscode-reborn-ai/commit/b408241ebc5863e3c6f99e493358e886da2ba523

File: package.nls.ru-ru.json
Comment (Christopher-Hayes) [inline package.nls.ru-ru.json:1]: Rename to `package.nls.ru.json`.

**Review by Christopher-Hayes — CHANGES_REQUESTED** [review]

**Comment by PeterDaveHello** [comment]
Sure, I can update the filename. Do you know any other locale needs this kind of change that I can do together?

File: package.nls.ru-ru.json
Comment (PeterDaveHello) [reply package.nls.ru-ru.json:1]: Done, thanks.

**Comment by Christopher-Hayes** [comment]
Sorry, took a little break from coding, I'll get this PR reviewed.

**Comment by Christopher-Hayes** [comment]
Looks good.

**Review by Christopher-Hayes — APPROVED** [review]

**Pull request merged**